lib/node_modules/@stdlib/blas/base/sspr/README.md

```diff
@@ -183,21 +183,60 @@ console.log( AP );
 ### Usage
 
 ```c
-TODO
+#include "stdlib/blas/base/sspr.h"
+#include "stdlib/blas/base/shared.h"
```

Comment:
@aman-095 This include should be removed from this code block and explicitly added to the subsequent code blocks.